validator/src/network/rest/routes/topology.rs

```diff
@@ -0,0 +1,12 @@
+use super::*;
+use crate::network::rest::models::presence;
+
+pub fn get(_req: &mut Request) -> IronResult<Response> {
+    let topology = presence::Topology {
+        mix_nodes: Vec::<presence::MixNode>::new(),
+        service_providers: Vec::<presence::ServiceProvider>::new(),
+        validators: Vec::<presence::Validator>::new(),
+    };
+    let response = serde_json::to_string_pretty(&topology).unwrap();
```

Comment:
Any reason we prettify the response rather than just use `to_string` ?

Reply:
I just like emitting pretty json wherever possible.